pxtblocks/code-validation/runValidatorPlanAsync.ts

```diff
@@ -36,4 +37,13 @@ namespace pxt.blocks {
             blockResults.insufficientBlocks.length === 0;
         return success;
     }
+
+    function runValidateBlockCommentsExist(usedBlocks: Blockly.Block[], inputs: BlockCommentExistsValidatorCheck): boolean {
```

Comment:
Are you saying to have something like the `blockCounts` object for this or just an optional `count` field? I though about using `blockCounts` but didn't because I wasn't sure how I would represent all of a block type in number form, but the optional field would work.

Reply:
Yeah, cool, that's what I was thinking about, too.